leaf-ui/rappid-extensions/IntermediateValues.js

```diff
@@ -6,7 +6,8 @@ var IntermediateValuesTable = Backbone.View.extend({
     model: joint.dia.BloomingGraph,
 
     template: ['<script type="text/template" id="item-template">',
-        '<div id="intermediateTable" class="intermT">',
+        '<div id="intermediateTable" class="intermT"; style="margin-left:110px";>',
```

Comment:
Is it possible to remove the `style` part and move the `margin-left` to the css class? I don't think `intermT` is being used for anything else so you could make the HTML a bit cleaner

Reply:
Okay, will do.